src/index.css

```
@@ -1,14 +1,153 @@
+    scrollbar-width: none;
+  }
+
+  .scrollbar-hide::-webkit-scrollbar {
+    display: none;
+  }
+
+  /* 滚动时显示滚动条，默认隐藏 - macOS 风格 */
+  .scrollbar-auto {
+    scrollbar-width: thin;
+    scrollbar-color: transparent transparent;
+    overflow: overlay;
```

Stylelint (17.7.0)

[error] Expected "overlay" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)